sdk/communication/azure-communication-common/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 ## 1.0.0-beta.4 (Unreleased)
 ### Breaking Changes
 - Renamed `CommunicationUserCredential` to `CommunicationTokenCredential`
+- Replaced constructor `CommunicationTokenCredential(TokenRefresher tokenRefresher, String initialToken,	boolean refreshProactively)` with `CommunicationTokenCredential(TokenRefreshOptions tokenRefreshOptions, String initialToken)`
+- Replaced constructor `CommunicationTokenCredential(TokenRefresher tokenRefresher)` with `CommunicationTokenCredential(TokenRefreshOptions tokenRefreshOptions)`
```

Comment:
Should we also include the breaking change of renaming CommunicationUser to CommunicationUserIdentifier in this file? Or do you plan to include that update in a different PR?

Reply:
Good point JP, I will need to include that in the other PR that has the renaming changes. This PR is about something different so I won't include it in here.